src/components/pages/Offers/Offer/Stocks/StocksGenericContainer.jsx

```diff
@@ -0,0 +1,12 @@
+import EventStockContainer from './EventStockContainer'
+import ThingStockContainer from './ThingStockContainer'
+
+const StocksGenericContainer = props => {
+  const isEvent = props.offer.isEvent
+  if (isEvent) {
+    return EventStockContainer(props)
+  }
+
+  return ThingStockContainer(props)
```

Comment:
```suggestion
  return isEvent ? EventStockContainer(props) : ThingStockContainer(props)
```

simple suggestion, à voir ce qui est le mieux :)

Reply:
Corrigé